### DISCUSSION TITLE
Enable Gzip compression

### DISCUSSION THREAD

**Discussion by informatikTirol** [directus/directus#5762] [Feature Requests]
Hi, 
a good feature would be gzip compression. 

Steps could be:  npm i compression --save

2) New .env var, e.g. GZIP_COMPRESSION="true"

in app.ts

```
const compression = require('compression');
...

if (env.GZIP_COMPRESSION === true) {
 app.use(compression());
}
...
```

**Comment by azrikahar** [comment]
Revisiting after nearly two years later, sharing 2 alternatives that comes to mind for future readers would be:

- using an [Init type hooks extension](https://docs.directus.io/extensions/hooks.html#init-events) to add said compression middleware
- delegating this responsibility to a reverse proxy

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
